web_transport/sdk/impl/web_transport_server_session.cc

```diff
@@ -171,6 +171,16 @@ const ConnectionStats& WebTransportServerSession::GetStats() {
 }
 
 void WebTransportServerSession::Close(uint32_t code, const char* reason) {
+  if (io_runner_->BelongsToCurrentThread()) {
+    return CloseOnCurrentThread(code, reason);
+  }
+  io_runner_->PostTask(
```

Comment:
I didn't see all other files. Does this mean the `Close` must be called before the destructor of `WebTransportServerSession`?
Otherwise, the destruction may happen before the execution of `CloseOnCurrentThread`, thus the `CloseOnCurrentThread` function will never be executed.

Reply:
Yes, `WebTransportServerSession` must not be destroyed. Currently, these objects are not cleaned up, so using plain `this` here is fine at this time. An issue https://github.com/open-webrtc-toolkit/owt-server/issues/1083 was opened in server repo for cleanup. I'll firstly handle close to make sure all objects are correctly converted to closed state. Then implementing deletions for these objects to avoid memory leaks.